baybe/surrogates/custom.py

```diff
@@ -220,3 +219,6 @@ def validate_compatibility(cls, searchspace: SearchSpace) -> None:
     #   properly. Potentially related to the conditional class definition, which should
     #   vanish as well.
     resolve_types(CustomONNXSurrogate)
+
+    # Block serialization of ONNX surrogates
+    converter.register_unstructure_hook(CustomONNXSurrogate, block_serialization_hook)
```

Comment:
Why is it necessary? This hook is already registered in `construct_custom_architecture`, why is it necessary to register it here again?

Reply:
@AVHopp: This is in fact wrong and and was still an artifact of the thing I discussed with you after the meeting yesterday. Also, the tests failed because of it. Have just removed one commit that was no longer necessary and now everything should be fine. With that, I think the PR should be ready now 🚀